src/libs/actions/User.ts

```diff
@@ -754,90 +746,107 @@ function setContactMethodAsDefault(newDefaultContactMethod) {
             },
         },
     ];
-    API.write('SetContactMethodAsDefault', {partnerUserID: newDefaultContactMethod}, {optimisticData, successData, failureData});
+
+    type SetContactMethodAsDefaultParam = {
+        partnerUserID: string;
+    };
+
+    const parameters: SetContactMethodAsDefaultParam = {
+        partnerUserID: newDefaultContactMethod,
+    };
+
+    API.write('SetContactMethodAsDefault', parameters, {
+        optimisticData,
+        successData,
+        failureData,
+    });
     Navigation.goBack(ROUTES.SETTINGS_CONTACT_METHODS.route);
 }
 
-/**
- * @param {String} theme
- */
-function updateTheme(theme) {
-    const optimisticData = [
+function updateTheme(theme: ValueOf<typeof CONST.THEME>) {
+    const optimisticData: OnyxUpdate[] = [
         {
             onyxMethod: Onyx.METHOD.SET,
             key: ONYXKEYS.PREFERRED_THEME,
             value: theme,
         },
     ];
 
-    API.write(
-        'UpdateTheme',
-        {
-            value: theme,
-        },
-        {optimisticData},
-    );
+    type UpdateThemeParam = {
+        value: string;
+    };
+
+    const parameters: UpdateThemeParam = {
+        value: theme,
+    };
+
+    API.write('UpdateTheme', parameters, {optimisticData});
 
     Navigation.navigate(ROUTES.SETTINGS_PREFERENCES);
 }
 
 /**
  * Sets a custom status
- *
- * @param {Object} status
- * @param {String} status.text
- * @param {String} status.emojiCode
  */
-function updateCustomStatus(status) {
-    API.write('UpdateStatus', status, {
-        optimisticData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-                value: {
-                    [currentUserAccountID]: {
-                        status,
-                    },
+function updateCustomStatus(status: CustomStatus) {
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.PERSONAL_DETAILS_LIST,
+            value: {
+                [currentUserAccountID]: {
+                    status: status.text,
```

Comment:
Why did you change from `status,` to `status: status.text,`?

Reply:
This could cause regression, our type maybe was wrong? Either way let's test it @fvlvte